setup.py

```diff
@@ -38,7 +38,7 @@
         'pennylane.plugins': [
             'qiskit.aer = pennylane_qiskit:AerQiskitDevice',
```

Comment:
Hey @carstenblank, I'm just curious - in the documentation, you mention `qiskit.basicaer` and `qiskit.ibmq`, but never `qiskit.aer`. Is this a working device that you can also use with this plugin?

Reply:
Hey @josh146, it is indeed possible and recommended to use the `qiskit.aer` device. The reason this wasn't documented was because of historic reasons. During the time when the qiskit team pulled out the aer from `qiskit-terra` it was dysfunctional. But all issues seem to have been fixed.
I am adding more documentation right now to the readme.